taxonomies/allergens.txt

```diff
@@ -366,7 +372,7 @@ en:sulphur dioxide and sulphites, sulphur dioxide, sulphites, sulfites
 ar:ثاني أكسيد الكبريت والكبريتات, ثاني أكسيد الكبريت, الكبريتات
 bg:серен диоксид и сулфити, серен диоксид, сулфити
 cs:oxid siřičitý a siřičitany, oxid siřičitý, siřičitany
-da:svovldioxid og sulfitter, svovldioxid, sulfitter
+da:svovldioxid og sulfitter, svovldioxid, sulfitter, natriumsulfit
```

Comment:
the other languages don't mention sodium sulphite (which is apparently E221). 
don't know if that's intentional/significant, but it stood out to me.
(also in sv:)

Reply:
I don't know if it belongs either, but I think I added it because it appears highlighted on ingredient lists. [example](https://se.openfoodfacts.org/product/7310751194518/ica-potatismos)

Reply:
Now that I think about it, you don't need to list it for languages like French because it's in separate words, i.e **sulfite** de sodium. OFF currently breaks the ingredient if you try to mark part of a compound word as an allergen.